docs/source/admin/index.rst

```diff
@@ -16,12 +16,14 @@ For medium-sized deployments (a handful of users), an AiiDAlab multi-user server
 Since deploying a multi-user server requires additional packages, such as `JupyterHub <https://jupyter.org/hub>`__, `DockerSpawner <https://github.com/jupyterhub/dockerspawner>`__, `Apache HTTP Server <https://www.apache.org/>`__, and `Docker <http://www.docker.com>`__, we provide the ``aiidalab-server`` `Ansible <https://www.ansible.com/>`__ role which automates the setup of the server.
 Please see the `corresponding git repository <https://github.com/aiidalab/ansible-role-aiidalab-server>`__ for more information.
 
+We also provide `instructions for deploy with microk8s in a single-server setup <https://github.com/aiidalab/aiidalab-microk8s-deploy#readme>`. This is a good option if you want to deploy AiiDAlab on a single machine, but expect a moderate amount of users (<50).
```

Comment:
I think there is a mistake here, if you checkout the rendering at https://aiidalab--393.org.readthedocs.build/en/393/admin/index.html#single-server-deployment (introduced by my suggestion, I did not realize this is a rst file and not markdown).

```suggestion
We also provide `instructions for deploy with microk8s in a single-server setup <https://github.com/aiidalab/aiidalab-microk8s-deploy#readme>`__. This is a good option if you want to deploy AiiDAlab on a single machine, but expect a moderate amount of users (<50).
```